src/CAREUI/display/Count.tsx

```diff
@@ -22,7 +22,7 @@ export default function CountBlock(props: Props) {
           <dt className="my-2 truncate text-sm font-semibold text-gray-700">
             {props.text}
           </dt>
-          {props.loading ? (
+          {props.loading || (!props.count && props.count !== 0) ? (
```

Comment:
wouldn't this be stuck in the loading state of the count is actually 0

we should not collace count to 0 if it's undefined while passing it to the component, and hide loading state when the count becomes defined

Reply:
Didn't understand,I am already checking if the count is not 0,do I still need to change the approach?
@sainak